lib/matplotlib/colorbar.py

```diff
@@ -1119,13 +1127,13 @@ def _mesh(self):
         else:
             y = self._proportional_y()
         xmid = np.array([0.5])
-        try:
+        if not self._scale == 'manual':
```

Comment:
```suggestion
        if self._scale != 'manual':
```

Makes the not-equal logic easier to read to my eye, but :man_shrugging: one way or the other.